src/store/actionCreators/userAuth.ts

```diff
@@ -1,23 +1,24 @@
-import { IUserAuth, setUser } from './user';
-import { UserActionType } from '@store/types/user';
 import { Dispatch } from 'redux';
 import axios, { AxiosResponse } from 'axios';
-import { API_URL } from '@constants/apiUrl';
 import * as cookies from '@core/cookies/cookies';
+import { API_URL } from '@constants/apiUrl';
+import { UserActionType } from '@store/types/user';
+import { IUserAuth, setUser } from './user';
 
 export const userAuth = () => {
   return async (dispatch: Dispatch<UserActionType | UserActionType>) => {
-    
     try {
-      const response: AxiosResponse<{ user: IUserAuth; token: string }> = await axios.get(
-        API_URL.USER_AUTHORIZATION,
-        { headers: { Authorization: `Bearer ${cookies.getCookie('token')}` } }
-      );
+      const response: AxiosResponse<{
+        user: IUserAuth;
+        tokens: { accessToken: string; refreshToken: string };
```

Comment:
why do we need tokens on the client side?

Reply:
fixed

Reply:
src/store/actionCreators/signIn.ts

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/23/files#diff-cbab2477fab901ef1b548728730561e91008a6f072bfdcef990bc91275396251